doc/developers/contributing.rst

```diff
@@ -621,27 +649,37 @@ the development version.
 ..
     packaging is not needed once setuptools starts shipping packaging>=17.0
 
-Building the documentation requires installing some additional packages::
+Building the documentation requires installing some additional packages:
+
+.. prompt:: bash $
 
     pip install sphinx sphinx-gallery numpydoc matplotlib Pillow pandas \
                 scikit-image packaging seaborn
```

Comment:
Please add sphinx-prompt to this list of doc build dependencies.

Reply:
The requested change has been added.